action.yml

```diff
@@ -24,6 +24,10 @@ inputs:
   files:
     description: "Newline-delimited list of path globs for asset files to upload"
     required: false
+  overwrite_files:
+    description: "Overwrite existing files with the same name. Defaults to false"
+    required: false
+    default: 'true'
```

Comment:
this is wrong

Reply:
Why?

Reply:
because it does not match with desc

<img width="685" alt="image" src="https://github.com/user-attachments/assets/cc72c93f-c9ed-419e-b62d-86066381072c">

Reply:
Fixed.